README.md

```diff
@@ -1,2 +1,176 @@
+<br>
+<img src="./vm.PNG">
+
+### RF Module
+
+Generally, an RF module is a small size electronic device, that is used to transmit or receive radio signals between two devices. The main application of the RF module is an embedded system to communicate with another device wirelessly.
+<br>
+<img src="./rf.PNG">
+
+### MQ-3 Alcohol Sensor
+
+The MQ-3 is Alcohol Sensitive Sensor that is often used in a breath analyzer.An alcohol sensor detects the attentiveness of alcohol gas in the air and an analog voltage is an output reading.
+<br>
+<img src="./as.PNG">
+
+### Force sensitive Resistor
+
+A force-sensing resistor is a material whose resistance changes when a force, pressure or mechanical stress is applied. It is utilised to know whether the helmet is being worn or not.
+<br>
+<img src="./fsr.PNG">
+
+### GPS Module
+
+The GPS receiver measures the distance to each satellite by the amount of time it takes to receive a transmitted signal. With distance measurements from a few more satellites, the receiver can determine a user's position and display it
+<br>
+<img src="./gps.PNG">
+
+### The microcontrollers used are:
+
+### Arduino UNO
+
+### NodeMCU
+
+## Software Aspect
+
+IoT is a concept that refers to all the objects that are now connected to the internet and how they can communicate with each other or with the people around them. 
+Iot architecture for our project is
+<br>
+<img src="./iotarchi.PNG" >
+<br>
+<br>
+<br>
+<img src="./dataflow.PNG">
+
+
+Arduino IoT cloud services act as a cloud-based hosting platform connecting the NodeMCU to the cloud.
+Wireless communication of messages to the rider's family are sent using PushBullet app.
+
+
+## Tech Stack
+
+## Cost Structure
+
+<table>
+<tr>
+<th>COMPONENTS</th>
+<th>COST(in Rs.)</th>
+</tr>
+<tr>
+    <td>Arduino UNO (QTY-2)
+    <td>
+</tr>
+<tr>
+    <td>NodeMCU ESP8266
+    <td>
+</tr>
+<tr>
+    <td>ADXL345 Accelerometer
+    <td>
+</tr>
+<tr>
+    <td>RF Transmitter and Receiver 433Mhz
+    <td>
+</tr>
+<tr>
+    <td>NEO-6M GPS Module
+    <td>
+</tr>
+<tr>
+    <td>Force Sensitive Resistor
+    <td>
+</tr>
+<tr>
+    <td>MQ3 Alcohol Sensor
+    <td>
+</tr>
+<tr>
+    <td>Jumper Wires
+    <td>
+</tr>
+<tr>
+    <td>Bike Helmet
+    <td>300
+</tr>
+<tr>
+    <td>Vibration Sensor module 
+    <td>
+</tr>
+</table>
+
+## Applications
+
+<ul>
+<li>Accident prevention and real time safety system
+<li>Rescue requests
+<li>Provision of help in time
+</ul>
+
+## Limitations
+
+<ul>
+<li>False alarms are possible
+<li>GPS module not implemented
+</ul>
+
+<br>
+
+## Future Improvements
+
+<ul>
+<li>Other sensors like heartbeat monitors, and oximeters can be introduced
+<li>Feedback mechanisms for the rider can be introduced
+<li>Can be designed for less power consumption
+<li>The microcontroller could be better placed so that it remains secure and safe from external stresses.
+</ul>
+
+<br>
+
+## Team Members
+
+<li> Bhawna Rana
```

Comment:
add links for GitHub profile also

Reply:
done

Reply:
add links for mentor's profile also. you can create a hyperlink in markdown by using:
```
[name](URL)
```